src/docs/guide/animations.md

```diff
@@ -7,4 +7,4 @@ order: 10
 show_guide: true
 ---
 
-Coming soon.
+Coming soon. See the detailed [animation docs](../core/animation.html) in the meantime.
```

Comment:
aframe-site-dev is ok. just want to link to relevant docs in the meantime, especially since the animation docs are already comprehensive